11/jdk/ubuntu/focal/entrypoint.sh

```diff
@@ -1,30 +1,89 @@
-#!/usr/bin/env bash
-# Sheband needs to be `bash`, see https://github.com/adoptium/containers/issues/415 for details
+#!/usr/bin/env sh
+# Converted to POSIX shell to avoid the need for bash in the image
```

Comment:
This is a problem for dotted environment variables, right?

Reply:
Yes and no, this hunk is based off https://github.com/adoptium/containers/commit/a6be406db8c418753541b409fcc06dde5db99308, where I missed that it only affected Alpine-based images, so I've definitely been a bit overzealous in converting the entrypoint script. I've checked this now and the only two images that do not support dotted variables in `/bin/sh` are now the Ubuntu-based images, which will need to have `/bin/bash` as their shebang. I'll see whether I can resolve this elegantly. It would certainly helped if we'd revived https://github.com/docker-library/official-images/pull/15668 to make sure it keeps working.

Reply:
Our deployments are broken now, because we rely on the dotted env variables with ubuntu images.
This lead me to this change.
Apparently the issue was not solved.

Reply:
There is https://github.com/adoptium/containers/pull/572 which addresses the issue. Still needs to be reviewed, approved and released.